bot/cogs/booster.py

```diff
@@ -21,15 +28,33 @@ async def on_connect(self) -> None:
     async def _monthly_booster(self) -> None:
         date = datetime.now(pytz.timezone('Asia/Jakarta'))
         if date.day == 1:
-            channel = self.bot.get_channel(ADMIN_CHANNEL_ID)
+            admin_channel = self.bot.get_channel(ADMIN_CHANNEL_ID)
+            tatsu_log_channel = self.bot.get_channel(TATSU_LOG_CHANNEL_ID)
             guild = self.bot.get_guild(GUILD_ID)
             role = guild.get_role(BOOSTER_ROLE_ID)
+            month = date.strftime("%B")
             members_content = ''
             for member in role.members:
+                await TatsuApi().add_score(member.id, 0, BOOSTER_MONTHLY_EXP)
+                embed = discord.Embed(
+                    title="<a:checklist:1077585402422112297> Score updated!",
+                    description=f"Successfully awarded `{BOOSTER_MONTHLY_EXP}` score to <@{member.id}>",
+                    timestamp=datetime.now(),
+                    colour=0x17A168,
+                )
+                embed.set_footer(
+                    text="Warnet",
+                    icon_url="https://cdn.discordapp.com/attachments/761684443915485184/1038313075260002365/warnet_logo_putih.png",
+                )
+                await tatsu_log_channel.send(embed=embed)
                 members_content += f"{member.id} "
+                sleep(1.5)
             buffer = io.BytesIO(members_content.encode('utf-8'))
-            file = discord.File(buffer, filename="honorary.txt")
-            await channel.send(file=file)
+            file = discord.File(buffer, filename=f"{month}_honorary.txt")
+            await admin_channel.send(
+                content=f"Exp Honorary Bulanan Sudah dibagikan, ANNOUNCEMENT! \n log honorary bulan {month}",
```

Comment:
I'm not fond of this sentence. Could you fix it? Especially that "ANNOUNCEMENT!"